python/tests/datasources/test_uhc_brfss.py

```diff
@@ -1,85 +1,341 @@
 TEST_DIR = os.path.join(THIS_DIR, os.pardir, "data", "uhc_brfss")
 
-GOLDEN_DATA = {
-    'race_and_ethnicity': os.path.join(TEST_DIR, 'uhc_test_output_race_and_ethnicity.json'),
-    'sex': os.path.join(TEST_DIR, 'uhc_test_output_sex.json'),
-    'age': os.path.join(TEST_DIR, 'uhc_test_output_age.json')}
+GOLDEN_DATA_RACE = os.path.join(TEST_DIR, 'uhc_test_output_race_and_ethnicity.json')
+GOLDEN_DATA_AGE = os.path.join(TEST_DIR, 'uhc_test_output_age.json')
+GOLDEN_DATA_SEX = os.path.join(TEST_DIR, 'uhc_test_output_sex.json')
 
 
 def get_test_data_as_df():
-    return pd.read_csv(os.path.join(TEST_DIR, 'uhc_test_input.csv'),
-                       dtype={"State Name": str,
-                              "Measure Name": str,
-                              "Value": float,
-                              })
+    df = pd.read_csv(os.path.join(TEST_DIR, 'uhc_test_input.csv'),
+                     dtype={"State Name": str,
+                            "Measure Name": str,
+                            "Value": float})
 
+    df_national = df.copy().reset_index(drop=True)
+    df_national['State Name'] = 'United States'
 
+    return pd.concat([df, df_national]).reset_index(drop=True)
+
+
+def get_race_pop_data_as_df_state():
+    return pd.read_csv(os.path.join(TEST_DIR, 'population_race.csv'), dtype=str)
+
+
+def get_age_pop_data_as_df_state():
+    return pd.read_csv(os.path.join(TEST_DIR, 'population_age.csv'), dtype=str)
+
+
+def get_sex_pop_data_as_df_state():
+    return pd.read_csv(os.path.join(TEST_DIR, 'population_sex.csv'), dtype=str)
+
+
+def get_race_pop_data_as_df_national():
+    df = pd.read_csv(os.path.join(TEST_DIR, 'population_race.csv'), dtype=str)
+    df[std_col.STATE_FIPS_COL] = '00'
+    df[std_col.STATE_NAME_COL] = 'United States'
+    return df
+
+
+def get_age_pop_data_as_df_national():
+    df = pd.read_csv(os.path.join(TEST_DIR, 'population_age.csv'), dtype=str)
+    df[std_col.STATE_FIPS_COL] = '00'
+    df[std_col.STATE_NAME_COL] = 'United States'
+    return df
+
+
+def get_sex_pop_data_as_df_national():
+    df = pd.read_csv(os.path.join(TEST_DIR, 'population_sex.csv'), dtype=str)
+    df[std_col.STATE_FIPS_COL] = '00'
+    df[std_col.STATE_NAME_COL] = 'United States'
+    return df
+
+
+EXPECTED_DTYPE = {
+    'state_name': str,
+    'state_fips': str,
+    "diabetes_per_100k": float,
+    "copd_per_100k": float,
+    "frequent_mental_distress_per_100k": float,
+    "depression_per_100k": float,
+    "suicide_per_100k": float,
+    "illicit_opioid_use_per_100k": float,
+    "non_medical_rx_opioid_use_per_100k": float,
+    "non_medical_drug_use_per_100k": float,
+    "excessive_drinking_per_100k": float,
+    "preventable_hospitalizations_per_100k": float,
+    "avoided_care_per_100k": float,
+    "chronic_kidney_disease_per_100k": float,
+    "cardiovascular_diseases_per_100k": float,
+    "asthma_per_100k": float,
+    "voter_participation_per_100k": float,
+    'brfss_population_pct': str
```

Comment:
is this supposed to be `str` or should it be `float`

Reply:
I'll try it with float